app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,380 @@
+                                  ng-model="strategyData[displayedParams].timeoutSeconds"
+                                  placeholder="600"
+                                  ng-pattern="/^\d+$/"
+                                  min="0"
+                                  select-on-focus
+                                  class="form-control"
+                                  aria-describedby="strategyTimeout">
+                              <span class="input-group-addon">seconds</span>
+                            </span>
+                            <div class="help-block" ng-attr-id="strategyTimeout">
+                              How long should deployment configuration retry to run the pod before times out.
+                            </div>
+                            <div ng-if="form.strategyTimeout.$invalid && form.strategyTimeout.$touched" class="has-error">
+                              <div ng-if="form.strategyTimeout.$error.number" class="help-block">
+                                Must be a number.
+                              </div>
+                              <div ng-if="form.strategyTimeout.$error.min" class="help-block">
+                                Delay can't be negative.
+                              </div>
+                              <span ng-if="form.strategyTimeout.$error.pattern && !form.strategyTimeout.$error.min" class="help-block">
+                                Must be a whole number.
+                              </span>
+                            </div>
+                          </div>
+
+                          <div ng-if="strategyData.type === 'Rolling'">
+                            <div ng-if="view.advancedStrategyOptions">
+                              <div class="form-group">
+                                <label for="updatePeriod">Update Period</label>
+                                <span class="input-group"
+                                   ng-class="{ 'has-error': form.updatePeriod.$invalid && form.updatePeriod.$touched }">
+                                  <input id="updatePeriod"
+                                      type="number"
+                                      placeholder="1"
+                                      name="updatePeriod"
+                                      ng-model="strategyData[displayedParams].updatePeriodSeconds"
+                                      ng-pattern="/^\d+$/"
+                                      min="0"
+                                      select-on-focus
+                                      class="form-control"
+                                      aria-describedby="updatePeriod">
+                                  <span class="input-group-addon">seconds</span>
+                                </span>
+                                <div class="help-block" ng-attr-id="updatePeriod">
+                                  Time to wait between retrying to run individual pod.
```

Comment:
should be right based on our api https://github.com/openshift/origin/blob/master/pkg/deploy/api/v1/types.go#L141